lib/cmds/update.ts

```diff
@@ -61,12 +61,14 @@ if (argv._[0] === 'update-run') {
   prog.printHelp();
 }
 
+let browserFile: BrowserFile;
 
 /**
  * Parses the options and downloads binaries if they do not exist.
  * @param options
  */
-function update(options: Options): void {
+function update(options: Options): q.IPromise<void> {
```

Comment:
Sorry, I think I might have misspoke:

`update` should still use `Promise<void>`, since it's return value is generated using `Promise.all`.

`promises` can (probably?) use the type `q.IPromise<void>[]`, which is better than `any[]`